src/main/riddl/Venues/venueTypeDefinitions.riddl

```diff
@@ -0,0 +1,37 @@
+type Status is any of {Draft, Open, Closed} briefly "Internal status of a Venue."
+
+type Info is {
+	orgId: OrganizationId briefly "Venues are defined by BaseOrgs. venueId and orgId make a composite key",
+  name: String,
+  address: Address,
+  isPrivate: Boolean briefly "Is this location accessible outside of the organization owning its venue?"
```

Comment:
You would need an organizationId to see if it falls outside of the organization. I have made several of these organization contexts, owners being organizations etc. But a thought just struck me this time (good thing it finally did). This idea of organization ownership is good. I think it is interesting to have a private location/venue too. But to enforce this private idea you would need to recurse down the organization hierarchy to see if a reserving org belongs to the owning org (a generational child). Let me know if this idea does not make sense.

Reply:
I see OrgId in the info block above, but will leave the comment to capture the idea of the generational child and private Venues.

Reply:
It's a good thing to point out. We would need to write a query on isPrivate connecting it to a projection based on parents & children of OrgId. I don't think that would be too difficult to build out though. Will mention it in the comment.